fiftyone/migrations/revisions/v0_6_2.py

```diff
@@ -0,0 +1,21 @@
+"""
+FiftyOne v0.6.2 revisions
+
+| Copyright 2017-2020, Voxel51, Inc.
+| `voxel51.com <https://voxel51.com/>`_
+|
+"""
+import pymongo as pm
```

Comment:
unused? (Or does importing this have side effects?)

Reply:
It will be used. Note that these migrations should not use `fiftyone` at all. A revision can't assume the same `fo` API will always be available.

So, the point is, by using `pymongo` only, which is essentially stable, avoids any issues with changes to `fiftyone`.